src/scripts/build.ts

```diff
@@ -82,7 +82,3 @@ export const build = async (
 
   return results;
 };
-
-if (require.main === module) {
-  build();
-}
```

Comment:
Doesn't removing this make `yarn build` a no-op?

Reply:
i moved it under `src`